include/revng/Model/Importer/DebugInfo/DwarfImporter.h

```diff
@@ -36,9 +36,7 @@ class DwarfImporter {
   TupleTree<model::Binary> &getModel() { return Model; }
 
 public:
-  void import(llvm::StringRef FileName, const ImporterOptions &Options);
-
-  void import(llvm::MemoryBuffer &Buffer,
+  void import(llvm::StringRef Filepath,
               const ImporterOptions &Options,
               llvm::StringRef Filename);
```

Comment:
I'm not happy with these arguments. The first and last seem redundant.

NOTETOSELF: review meaning of `Filename`.

Reply:
Yes, but passing `/tmp/binaries-container-file-73ad2c` for the latter is not the right thing to do.

Reply:
Fixed now, there's `/bin/bash` in the `BinaryIdentifier.Name` now